### DISCUSSION TITLE
anyway to disable RTSP?

### DISCUSSION THREAD

**Discussion by letrain02** [mrlt8/docker-wyze-bridge#1248] [Q&A]
i don't use RTSP (i do use snapshots/timelapse not sure if that would be affected). With the new PCMU/8000 codec is constantly running when  i have audio enabled even though i'm only using webrtc. can i disable re-encoding even though i dont need it for webrtc?

[back-lot] 📡 Getting 30kb/s SD stream (H264/20fps) via LAN mode (WiFi: 64%) FW: 4.50.4.9222 🔒
[back-lot] 🔊 Audio Enabled [Source=ALAW/16,000Hz]
[back-lot] [AUDIO] Re-Encode for RTSP compatibility [codec_out='pcm_mulaw']

**Comment by mrlt8** [comment]
We're creating an RTSP stream as the source for MediaMTX so I don't think you can disable RTSP.

As for `Re-Encode for RTSP compatibility` it's technically more for WebRTC as ALAW/16,000Hz is supported in RTSP but WebRTC only supports up to 8,000Hz because of chrome, so we're just re-encoding all audio to be RTSP/WebRTC compatible.

You can see the weird combo of audio formats in the MediaMTX table:
https://github.com/bluenviron/mediamtx

Edit: you _can_ disable the ports for RTSP if you're not using it.
